cloud/aws/terraform/common/s3notif/README.md

```diff
@@ -0,0 +1,11 @@
+# Notification stream of S3 events
+
+This modules forwards EventBridge notifications for new objects in a source
+`bucket_name` to a target event bus `target_bus_arn`.
+
+We prefer using EventBridge rather than the legacy lambda notification system
+because its pub/sub architecture allows us to subscribe to the notification feed
+without disturbing existing notification configurations on the source bucket.
+
+Note that for this module to work, EventBridge must be enabled on the source
+bucket. This can be achieved manually or using Terragrunt hooks.
```

Comment:
it cannot be done automatically in Terraform because of the way the API works: https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketNotificationConfiguration.html. If you use terraform, you will erase all existing notification settings on this bucket, which is not something we want as the assumption of this module is that the bucket is not owned. Let me clarify this in the doc.

Reply:
asserting that eventbridge is enabled might be a bit tricky, and I am not sure we would really want to block the deployment if it's not. I think we can leave it to the external user of the module to take care of this.